File: dotnet-maui/ReservationsDotnetMaui/Models/Office.cs
namespace ReservationsDotnetMaui.Models;

public class Office
{
    public string Name { get; set; } = string.Empty;
    public string Id { get; set; } = string.Empty;
    
    public static List<Office> GetOffices()
    {
        return new List<Office>
        {
            new Office { Id = "la", Name = "Los Angeles" },
            new Office { Id = "burbank", Name = "Burbank" },
            new Office { Id = "glendale", Name = "Glendale" },
            new Office { Id = "losfeliz", Name = "Los Feliz" },
            new Office { Id = "hollywood", Name = "Hollywood" }
        };
    }
}

File: flutter/flutterapp/lib/models/office.dart
class Office {
  const Office({required this.id, required this.name});

  final String id;
  final String name;

  static List<Office> getOffices() {
    return const [
      Office(id: 'la', name: 'Los Angeles'),
      Office(id: 'burbank', name: 'Burbank'),
      Office(id: 'glendale', name: 'Glendale'),
      Office(id: 'losfeliz', name: 'Los Feliz'),
      Office(id: 'hollywood', name: 'Hollywood'),
    ];
  }
}
